packages/rancher-istio/scripts/version_update.go

```diff
@@ -0,0 +1,60 @@
+package main
```

Comment:
Is bringing in source code to the charts repo intentional?

Reply:
yes, this is a script for this specific chart update all the versions for istio and kiali easily - its just a nice to have script

Reply:
This won't end up in helm package right ? Want to confirm that.

Reply:
it wont since its at the same level as the charts directory, and the rancher-istio/charts is what gets packaged